pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -408,34 +412,32 @@ one = 'Name of the Kafka instance you want to view'
 
 [kafka.describe.flag.bootstrapserver]
 description = 'Description for the --bootstrap-server flag'
-one = 'If specified, only bootstrap server host of the Kafka instance will be displayed'
+one = 'If specified, only the bootstrap server host of the Kafka instance will be displayed'
 
 [kafka.describe.bootstrapserver.not.available]
 one = 'Kafka instance "{{.Name}}" does not have a bootstrap server URL.'
 
 [kafka.list.cmd.shortDescription]
 description = "Short description for command"
-one = "List all Apache Kafka instances"
+one = "List all Kafka instances"
 
 [kafka.list.cmd.longDescription]
 description = "Long description for command"
 one = '''
-List all Apache Kafka instances.
+List all Kafka instances.
 
-This command will provide a high level view of all Kafka instances.
-The fields displayed are: ID, Name, Owner, Status, Cloud Provider, Region.
-Use the describe command to view all fields for a specific instance.
+By default, this command lists the Kafka instances in a table, showing the ID, name, owner, status, cloud provider, and region.
 
-The instances are displayed by default in a table, but can also be displayed as JSON or YAML.
```

Comment:
We should keep the text about JSON and YAML to be consistent with help text of other list commands.

Reply:
I added this back in.